gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,6 @@
+#Sat Jul 31 23:05:01 CEST 2021
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.0.2-all.zip
```

Comment:
Please don't change the gradle distribution here

Reply:
Revert to old Settings